content/automate/ManageIQ/Infrastructure/VM/Transform/Import.class/__methods__/list_driver_isos.rb

```diff
@@ -21,7 +21,7 @@ def main
                       values_hash[nil] = '-- no ISO datastore for provider --'
                     else
                       values_hash[nil] = '-- select drivers ISO from list --'
-                      provider.iso_datastore.iso_images.pluck(:name).grep(/toolsSetup/).each do |iso|
+                      provider.iso_datastore.iso_images.pluck(:name).grep(/tools.*setup/i).each do |iso|
```

Comment:
@matobet Is this condition too loose for the lookup?  Where do the `iso_image` names come from, does we have control over them?